<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>miaDash • miaDash</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet">
<meta property="og:title" content="miaDash">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">miaDash</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.4</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/miaDash.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/microbiome/miaDash/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>miaDash</h1>
                        <h4 data-toc-skip class="author">Giulio
Benedetti</h4>
            <address class="author_afil">
      University of
Turku<br><a class="author_email" href="mailto:#"></a><a href="mailto:giulio.benedetti@utu.fi" class="email">giulio.benedetti@utu.fi</a>
      </address>
                  
            <h4 data-toc-skip class="date">9 December 2024</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/microbiome/miaDash/blob/devel/vignettes/miaDash.Rmd" class="external-link"><code>vignettes/miaDash.Rmd</code></a></small>
      <div class="hidden name"><code>miaDash.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<p>This notebook provides a practical introduction to the Microbiome
Analysis Dashboard (miaDash), an interactive app to analyse and explore
microbiome data. Feel free to try it online at <a href="miadash-microbiome.2.rahtiapp.fi/">this address</a> with your data
or one of the ready-to-use example datasets. Here, its usage and
functionality are described in more detail.</p>
<div class="section level3">
<h3 id="motivation">Motivation<a class="anchor" aria-label="anchor" href="#motivation"></a>
</h3>
<p>Most of the tools available for microbiome data analysis require some
knowledge of programming. This represents a burden for practitioners
more interested in getting results than learning how to program. To this
end, miaDash aims to make microbiome analysis accessible to anyone who
needs it, with or without any computational skills.</p>
<p>As a word of caution, while the app removes the burden of
programming, it is still critical to understand the nature of microbiome
data and how it can be analysed. Such knowledge can be acquired from the
online book <a href="https://microbiome.github.io/OMA/" class="external-link">Orchestrating
Microbiome Analysis</a> (OMA) and several other independent resources.
The following section presents what is currently possible in the
app.</p>
</div>
<div class="section level3">
<h3 id="interface">Interface<a class="anchor" aria-label="anchor" href="#interface"></a>
</h3>
<p>The interface is divided into five tabs that reflect the steps of a
typical microbiome analysis pipeline. First, the dataset of interest can
be uploaded through the Import tab, where several data types and file
formats are supported. Alternatively, one of the available example
datasets can be used for practice. Second, a set of operations can be
applied to the dataset through the Manipulate tab, which include methods
for subsetting features by prevalence, agglomerating by taxonomic level
and transforming assays. Third, the dataset can be analysed through the
Estimate tab, which provide common techniques to quantify alpha and beta
diversity. Finally, results can be explored through the Visualise tab,
where an interactive explorer can be launched with a customisable set of
panels that illustrate different aspects of the data.</p>
</div>
</div>
<div class="section level2">
<h2 id="tutorial">Tutorial<a class="anchor" aria-label="anchor" href="#tutorial"></a>
</h2>
<p>The app can be used online or locally, depending on resource
availability and size of the dataset to analyse. In general, the online
version is freely available, so that data of any type can be tested
there. However, running the app locally might be a better option for
larger datasets (&gt; 1000 features). In this case, you may also
consider subsetting and/or agglomerating the data.</p>
<div class="section level3">
<h3 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h3>
<p>If you decided to run the app locally, make sure to have R installed
in your machine and execute the following command:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"miaDash"</span><span class="op">)</span></span></code></pre></div>
<p>Once the package is successfully installed, you should have access to
the development version of miaDash.</p>
</div>
<div class="section level3">
<h3 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h3>
<p>This section shows how to get started with miaDash. If you are using
it locally, run the next code chunk to launch the app. Otherwise, you
can skip it.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># Import miaDash</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/microbiome/miaDash" class="external-link">miaDash</a></span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Instantiate app</span></span>
<span><span class="va">app</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/miaDash.html">miaDash</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Launch miaDash</span></span>
<span><span class="kw">if</span> <span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/interactive.html" class="external-link">interactive</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>  <span class="fu">shiny</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/shiny/man/runApp.html" class="external-link">runApp</a></span><span class="op">(</span><span class="va">app</span><span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
<p>As described in Section <a href="#interface">1.2</a>, The dashboard
consists of five different windows with tools to import, manipulate,
analyse and visualise the dataset of choice. After launching the app, it
appears as follows:</p>
<p><img src="screenshots%2Fdashboard.png" width="100%"></p>
<p>At first, the variety of options might feel intimidating, so you can
click on the question mark on the top right to receive a short tour of
the windows available in the app.</p>
<p>Once the dataset was imported and analysed according to your
objective, you can choose which visualisations to use from the Visualise
window and press the button “Launch iSEEtree” to create and customise
the plots. After adjusting the parameters of the different panels, the
app might look something like this:</p>
<p><img src="screenshots%2FiSEEtree.png" width="100%"></p>
<p>As before, you can click on the question mark on the top right to
receive a tour of the panels and their parameters. The best way to get
familiar with the interface is to experiment with the parameters below
each panel.</p>
</div>
</div>
<div class="section level2">
<h2 id="resources">Resources<a class="anchor" aria-label="anchor" href="#resources"></a>
</h2>
<div class="section level3">
<h3 id="citation">Citation<a class="anchor" aria-label="anchor" href="#citation"></a>
</h3>
<p>We hope that miaDash will be useful for your research. Please use the
following information to cite the package and the overall approach.
Thank you!</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Citation info</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/citation.html" class="external-link">citation</a></span><span class="op">(</span><span class="st">"miaDash"</span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in citation("miaDash"): could not determine year for 'miaDash' from</span></span>
<span><span class="co">#&gt; package DESCRIPTION file</span></span>
<span><span class="co">#&gt; To cite package 'miaDash' in publications use:</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   Benedetti G, Lahti L (????). _miaDash: Shiny app for interactive</span></span>
<span><span class="co">#&gt;   analysis and exploration of microbiome data_. R package version</span></span>
<span><span class="co">#&gt;   0.99.4, &lt;https://github.com/microbiome/miaDash&gt;.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; A BibTeX entry for LaTeX users is</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   @Manual{,</span></span>
<span><span class="co">#&gt;     title = {miaDash: Shiny app for interactive analysis and exploration of microbiome data},</span></span>
<span><span class="co">#&gt;     author = {Giulio Benedetti and Leo Lahti},</span></span>
<span><span class="co">#&gt;     note = {R package version 0.99.4},</span></span>
<span><span class="co">#&gt;     url = {https://github.com/microbiome/miaDash},</span></span>
<span><span class="co">#&gt;   }</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="background-knowledge">Background Knowledge<a class="anchor" aria-label="anchor" href="#background-knowledge"></a>
</h3>
<p>miaDash originates from the joint effort of the R/Bioconductor
community. It is mainly based on the following software:</p>
<ul>
<li>R (R Core Team, 2024), statistical programming language</li>
<li>
<em><a href="https://bioconductor.org/packages/3.21/mia" class="external-link">mia</a></em> (Borman,
Ernst, Shetty, and Lahti, 2024), framework for microbiome data
analysis</li>
<li>
<em><a href="https://bioconductor.org/packages/3.21/iSEEtree" class="external-link">iSEEtree</a></em>
(Benedetti and Lahti, 2024), TreeSummarizedExperiment interactive
explorer</li>
<li>
<em><a href="https://bioconductor.org/packages/3.21/iSEE" class="external-link">iSEE</a></em>
(Rue-Albrecht, Marini, Soneson, and Lun, 2018), SummarizedExperiment
interactive explorer</li>
<li>
<em><a href="https://bioconductor.org/packages/3.21/TreeSummarizedExperiment" class="external-link">TreeSummarizedExperiment</a></em>
(Huang, Soneson, Ernst, Rue-Albrecht, Yu, Hicks, and Robinson, 2021), S4
container for hierarchical data</li>
<li>
<em><a href="https://CRAN.R-project.org/package=shiny" class="external-link">shiny</a></em> (Chang,
Cheng, Allaire, Sievert, Schloerke, Xie, Allen, McPherson, Dipert, and
Borges, 2024), web app development in R</li>
</ul>
</div>
<div class="section level3">
<h3 id="help">Help<a class="anchor" aria-label="anchor" href="#help"></a>
</h3>
<p>You can reach us by one of the communication channels listed <a href="https://microbiome.github.io/OMA/docs/devel/pages/support.html" class="external-link">here</a>.
We are happy to receive questions, suggestions as well as contributions.
For the last point, check the <a href="https://microbiome.github.io/mia/CONTRIBUTING.html" class="external-link">contributor
guidelines</a>.</p>
</div>
</div>
<div class="section level2">
<h2 id="reproducibility">Reproducibility<a class="anchor" aria-label="anchor" href="#reproducibility"></a>
</h2>
<p>R session information:</p>
<pre><code><span><span class="co">#&gt; R Under development (unstable) (2024-12-04 r87420)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 24.04.1 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.26.so;  LAPACK version 3.12.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; time zone: UTC</span></span>
<span><span class="co">#&gt; tzcode source: system (glibc)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods   base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt;  [1] miaDash_0.99.4              iSEE_2.19.1                 SingleCellExperiment_1.29.1 SummarizedExperiment_1.37.0</span></span>
<span><span class="co">#&gt;  [5] Biobase_2.67.0              GenomicRanges_1.59.1        GenomeInfoDb_1.43.2         IRanges_2.41.2             </span></span>
<span><span class="co">#&gt;  [9] S4Vectors_0.45.2            BiocGenerics_0.53.3         generics_0.1.3              MatrixGenerics_1.19.0      </span></span>
<span><span class="co">#&gt; [13] matrixStats_1.4.1           RefManageR_1.4.0            BiocStyle_2.35.0           </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;   [1] fs_1.6.5                        DirichletMultinomial_1.49.0     fontawesome_0.5.3              </span></span>
<span><span class="co">#&gt;   [4] lubridate_1.9.4                 httr_1.4.7                      RColorBrewer_1.1-3             </span></span>
<span><span class="co">#&gt;   [7] doParallel_1.0.17               tools_4.5.0                     backports_1.5.0                </span></span>
<span><span class="co">#&gt;  [10] utf8_1.2.4                      R6_2.5.1                        DT_0.33                        </span></span>
<span><span class="co">#&gt;  [13] vegan_2.6-8                     lazyeval_0.2.2                  mgcv_1.9-1                     </span></span>
<span><span class="co">#&gt;  [16] rhdf5filters_1.19.0             GetoptLong_1.0.5                permute_0.9-7                  </span></span>
<span><span class="co">#&gt;  [19] withr_3.0.2                     gridExtra_2.3                   cli_3.6.3                      </span></span>
<span><span class="co">#&gt;  [22] textshaping_0.4.1               shinyjs_2.1.0                   sandwich_3.1-1                 </span></span>
<span><span class="co">#&gt;  [25] slam_0.1-55                     sass_0.4.9                      mvtnorm_1.3-2                  </span></span>
<span><span class="co">#&gt;  [28] pkgdown_2.1.1                   systemfonts_1.1.0               yulab.utils_0.1.8              </span></span>
<span><span class="co">#&gt;  [31] foreign_0.8-87                  scater_1.35.0                   decontam_1.27.0                </span></span>
<span><span class="co">#&gt;  [34] rstudioapi_0.17.1               gridGraphics_0.5-1              shape_1.4.6.1                  </span></span>
<span><span class="co">#&gt;  [37] dplyr_1.1.4                     rbiom_1.0.3                     Matrix_1.7-1                   </span></span>
<span><span class="co">#&gt;  [40] biomformat_1.35.0               ggbeeswarm_0.7.2                fansi_1.0.6                    </span></span>
<span><span class="co">#&gt;  [43] DECIPHER_3.3.1                  abind_1.4-8                     lifecycle_1.0.4                </span></span>
<span><span class="co">#&gt;  [46] yaml_2.3.10                     rhdf5_2.51.0                    SparseArray_1.7.2              </span></span>
<span><span class="co">#&gt;  [49] grid_4.5.0                      promises_1.3.2                  crayon_1.5.3                   </span></span>
<span><span class="co">#&gt;  [52] shinydashboard_0.7.2            miniUI_0.1.1.1                  lattice_0.22-6                 </span></span>
<span><span class="co">#&gt;  [55] beachmat_2.23.4                 pillar_1.9.0                    knitr_1.49                     </span></span>
<span><span class="co">#&gt;  [58] ComplexHeatmap_2.23.0           rjson_0.2.23                    boot_1.3-31                    </span></span>
<span><span class="co">#&gt;  [61] lpSolve_5.6.22                  codetools_0.2-20                glue_1.8.0                     </span></span>
<span><span class="co">#&gt;  [64] ggfun_0.1.8                     data.table_1.16.4               MultiAssayExperiment_1.33.1    </span></span>
<span><span class="co">#&gt;  [67] vctrs_0.6.5                     png_0.1-8                       treeio_1.31.0                  </span></span>
<span><span class="co">#&gt;  [70] gtable_0.3.6                    cachem_1.1.0                    xfun_0.49                      </span></span>
<span><span class="co">#&gt;  [73] S4Arrays_1.7.1                  mime_0.12                       tidygraph_1.3.1                </span></span>
<span><span class="co">#&gt;  [76] iterators_1.0.14                bluster_1.17.0                  nlme_3.1-166                   </span></span>
<span><span class="co">#&gt;  [79] ggtree_3.15.0                   SnowballC_0.7.1                 bslib_0.8.0                    </span></span>
<span><span class="co">#&gt;  [82] irlba_2.3.5.1                   vipor_0.4.7                     rpart_4.1.23                   </span></span>
<span><span class="co">#&gt;  [85] colorspace_2.1-1                DBI_1.2.3                       Hmisc_5.2-1                    </span></span>
<span><span class="co">#&gt;  [88] nnet_7.3-19                     tidyselect_1.2.1                compiler_4.5.0                 </span></span>
<span><span class="co">#&gt;  [91] htmlTable_2.4.3                 BiocNeighbors_2.1.1             xml2_1.3.6                     </span></span>
<span><span class="co">#&gt;  [94] desc_1.4.3                      DelayedArray_0.33.3             colourpicker_1.3.0             </span></span>
<span><span class="co">#&gt;  [97] bookdown_0.41                   checkmate_2.3.2                 scales_1.3.0                   </span></span>
<span><span class="co">#&gt; [100] stringr_1.5.1                   digest_0.6.37                   minqa_1.2.8                    </span></span>
<span><span class="co">#&gt; [103] rmarkdown_2.29                  miaViz_1.15.2                   XVector_0.47.0                 </span></span>
<span><span class="co">#&gt; [106] htmltools_0.5.8.1               pkgconfig_2.0.3                 iSEEtree_1.1.0                 </span></span>
<span><span class="co">#&gt; [109] base64enc_0.1-3                 bibtex_0.5.1                    lme4_1.1-35.5                  </span></span>
<span><span class="co">#&gt; [112] sparseMatrixStats_1.19.0        listviewer_4.0.0                fastmap_1.2.0                  </span></span>
<span><span class="co">#&gt; [115] rlang_1.1.4                     GlobalOptions_0.1.2             htmlwidgets_1.6.4              </span></span>
<span><span class="co">#&gt; [118] UCSC.utils_1.3.0                shiny_1.9.1                     DelayedMatrixStats_1.29.0      </span></span>
<span><span class="co">#&gt; [121] farver_2.1.2                    jquerylib_0.1.4                 zoo_1.8-12                     </span></span>
<span><span class="co">#&gt; [124] jsonlite_1.8.9                  BiocParallel_1.41.0             tokenizers_0.3.0               </span></span>
<span><span class="co">#&gt; [127] BiocSingular_1.23.0             magrittr_2.0.3                  Formula_1.2-5                  </span></span>
<span><span class="co">#&gt; [130] scuttle_1.17.0                  GenomeInfoDbData_1.2.13         ggplotify_0.1.2                </span></span>
<span><span class="co">#&gt; [133] patchwork_1.3.0                 Rhdf5lib_1.29.0                 munsell_0.5.1                  </span></span>
<span><span class="co">#&gt; [136] Rcpp_1.0.13-1                   ape_5.8                         ggnewscale_0.5.0               </span></span>
<span><span class="co">#&gt; [139] viridis_0.6.5                   stringi_1.8.4                   rintrojs_0.3.4                 </span></span>
<span><span class="co">#&gt; [142] ggraph_2.2.1                    zlibbioc_1.53.0                 MASS_7.3-61                    </span></span>
<span><span class="co">#&gt; [145] plyr_1.8.9                      mediation_4.5.0                 parallel_4.5.0                 </span></span>
<span><span class="co">#&gt; [148] ggrepel_0.9.6                   Biostrings_2.75.1               graphlayouts_1.2.1             </span></span>
<span><span class="co">#&gt; [151] splines_4.5.0                   circlize_0.4.16                 igraph_2.1.2                   </span></span>
<span><span class="co">#&gt; [154] reshape2_1.4.4                  ScaledMatrix_1.15.0             evaluate_1.0.1                 </span></span>
<span><span class="co">#&gt; [157] tidytext_0.4.2                  RcppParallel_5.1.9              BiocManager_1.30.25            </span></span>
<span><span class="co">#&gt; [160] nloptr_2.1.1                    foreach_1.5.2                   tweenr_2.0.3                   </span></span>
<span><span class="co">#&gt; [163] httpuv_1.6.15                   tidyr_1.3.1                     purrr_1.0.2                    </span></span>
<span><span class="co">#&gt; [166] polyclip_1.10-7                 clue_0.3-66                     ggplot2_3.5.1                  </span></span>
<span><span class="co">#&gt; [169] ggforce_0.4.2                   rsvd_1.0.5                      xtable_1.8-4                   </span></span>
<span><span class="co">#&gt; [172] tidytree_0.4.6                  janeaustenr_1.0.0               later_1.4.1                    </span></span>
<span><span class="co">#&gt; [175] viridisLite_0.4.2               ragg_1.3.3                      tibble_3.2.1                   </span></span>
<span><span class="co">#&gt; [178] aplot_0.2.3                     memoise_2.0.1                   beeswarm_0.4.0                 </span></span>
<span><span class="co">#&gt; [181] cluster_2.1.6                   TreeSummarizedExperiment_2.15.0 timechange_0.3.0               </span></span>
<span><span class="co">#&gt; [184] shinyWidgets_0.8.7              shinyAce_0.4.3                  mia_1.15.6</span></span></code></pre>
</div>
<div class="section level2">
<h2 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<p>
<a id="bib-benedetti2024iseetree"></a><a href="#cite-benedetti2024iseetree">[1]</a><cite>
G. Benedetti and L. Lahti. <em>iSEEtree: Interactive visualisation for
microbiome data</em>. R package version 1.1.0. 2024. DOI:
<a href="https://doi.org/10.18129/B9.bioc.iSEEtree" class="external-link">10.18129/B9.bioc.iSEEtree</a>.
URL:
<a href="https://bioconductor.org/packages/iSEEtree" class="external-link">https://bioconductor.org/packages/iSEEtree</a>.</cite>
</p>
<p>
<a id="bib-borman2024microbiome"></a><a href="#cite-borman2024microbiome">[2]</a><cite>
T. Borman, F. Ernst, S. Shetty, et al. <em>mia: Microbiome
analysis</em>. R package version 1.15.6. 2024. DOI:
<a href="https://doi.org/10.18129/B9.bioc.mia" class="external-link">10.18129/B9.bioc.mia</a>.
URL:
<a href="https://bioconductor.org/packages/mia" class="external-link">https://bioconductor.org/packages/mia</a>.</cite>
</p>
<p>
<a id="bib-chang2024shiny"></a><a href="#cite-chang2024shiny">[3]</a><cite>
W. Chang, J. Cheng, J. Allaire, et al. <em>shiny: Web Application
Framework for R</em>. R package version 1.9.1. 2024. DOI:
<a href="https://doi.org/10.32614/CRAN.package.shiny" class="external-link">10.32614/CRAN.package.shiny</a>.
URL:
<a href="https://CRAN.R-project.org/package=shiny" class="external-link">https://CRAN.R-project.org/package=shiny</a>.</cite>
</p>
<p>
<a id="bib-huang2021treesummarizedexperiment"></a><a href="#cite-huang2021treesummarizedexperiment">[4]</a><cite>
R. Huang, C. Soneson, F. G. Ernst, et al. “TreeSummarizedExperiment: a
S4 class for data with hierarchical structure”. In:
<em>F1000Research</em> 9 (2021), p. 1246. URL:
<a href="https://f1000research.com/articles/9-1246" class="external-link">https://f1000research.com/articles/9-1246</a>.</cite>
</p>
<p>
<a id="bib-2024language"></a><a href="#cite-2024language">[5]</a><cite>
R Core Team. <em>R: A Language and Environment for Statistical
Computing</em>. R Foundation for Statistical Computing. Vienna, Austria,
2024. URL:
<a href="https://www.R-project.org/" class="external-link">https://www.R-project.org/</a>.</cite>
</p>
<p>
<a id="bib-ruealbrecht2018isee"></a><a href="#cite-ruealbrecht2018isee">[6]</a><cite>
K. Rue-Albrecht, F. Marini, C. Soneson, et al. “iSEE: Interactive
SummarizedExperiment Explorer”. In: <em>F1000Research</em> 7
(Jun. 2018), p. 741. DOI:
<a href="https://doi.org/10.12688/f1000research.14966.1" class="external-link">10.12688/f1000research.14966.1</a>.</cite>
</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Giulio Benedetti, Leo Lahti.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

      </footer>
</div>






  </body>
</html>
